scripts/kubectl-kubestellar-ensure-wmw

```diff
@@ -70,20 +72,57 @@ esac
 
 set -e
 
-original_ws=$(kubectl ws "${kubectl_flags[@]}" . --short)
+IFS=':' read -ra ws_array <<< "$wmw_name"
+first_item="true"
+parent=""
+kubectl config use-context $space_mgt
 
-if [ "$original_ws" != root ]; then
-    kubectl ws "${kubectl_flags[@]}" ..
-    if kubectl "${kubectl_flags[@]}" get workspaces.tenancy.kcp.io "$wmw_name" &> /dev/null; then
-	echo "Warning: parent has a child with the given WMW name; are you sure the right workspace is current?" >&2
-	sleep 15 # give a chance for ^C
-    fi
-    kubectl ws "${kubectl_flags[@]}" -
-fi
+create_space() {
```

Comment:
I do not understand the logic here.  If users ask for paths `root:foo:baz` and `root:bar:baz` then they will be thinking of a tree with five vertices, including two leaves --- but that is not what they will get.

Reply:
The `create_space` func is called from the `for` loop for each part of the path except `root`. So if user ask for `root:my-org:wmw-c` , the script will create Space for `my-org` and for `wmw-c`. Than `wmw-c` workspace will be created under `my-org` workspace.

Reply:
Correct, we currently don't support this because the space name is unique. We can return an error to the user if the leave space already exit, and we will open an issue to support full hierarchy.
We don't have to deal with paths. We can create the parent first and then run the script to create a wmw.
WDYT?